src/index.js

```diff
@@ -64,6 +70,8 @@ function editTask(li) {
 }
 
 function renderTasks() {
+  todoList.innerHTML = ''; // Clear the existing list before rendering
+
   savedTasks.forEach((task, index) => {
```

Comment:
- [ ] Great job on saving/loading the list to/from the localStorage however as suggested by the previous reviewer please make sure that when some boxes are marked completed that particular task status also updates in the local storage and when the page is reloaded the tasks that were marked completed stay marked completed